src/backend/common/Forward.h

```diff
@@ -64,6 +68,9 @@ namespace backend {
     static constexpr uint32_t kMaxVertexAttributes = 16u;
     static constexpr uint32_t kMaxVertexInputs = 16u;
     static constexpr uint32_t kNumStages = 3;
+    static constexpr uint32_t kMaxTextureViewsPerFramebuffer = 16u;
```

Comment:
Is there a practical reason for this, like a Vulkan limitation? Otherwise Framebuffer could require the renderpass to be given before the texture views, and resize a vector.